plugins/source/azure/codegen/recipes/web.go

```diff
@@ -53,10 +53,21 @@ func Web() []Resource {
 		mockDefinitionType:       "PublishingProfiles",
 	}
 
+	var functionsResource = resourceDefinition{azureStruct: &web.FunctionEnvelope{},
```

Comment:
Changes in this file are manually generated (adding functions as a relation of apps)